### PR TITLE
DRAFT: imgdata2hdf5, tool to convert raw img data to hdf5

### PR DESCRIPTION
**Pull request by jinevening**
On going draft to land imgdata2hdf5

ONE-DCO-1.0-Signed-off-by: Hyukjin Jeong <hj1.jeong@samsung.com>

---
Related to #3213

### CODE REVIEW

**Comment by jinevening** [comment]
@seanshpark This is just a draft. If interface is changed (discussed in #3213), I will update the code (changing interface may not take too much time).

**Comment by jinevening** [comment]
@seanshpark I've changed the interface to the SNPE style (using only data list file). PTAL :)

File: compiler/imgdata2hdf5/README.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# imgdata2hdf5
+
+_imgdata2hdf5_ is a tool to convert raw image data (assumed to be pre-processed) to an hdf5 file.
+
+## Prerequisite
+- Raw image data pre-processed for the corresponding DNN model
+- List of data to convert (saved in the text file)
+- Python installed with _numpy_ and _h5py_ (ex: pip install numpy h5py)
```
Comment (seanshpark) [inline compiler/imgdata2hdf5/README.md:8]: as install of `one-cmds` will provide a script to prepare a virtual environment, you can guide to use that.
- how-to-prepare-virtualenv.txt
- this fil will be installed in `docs` folder. refer #3236

Comment (jinevening) [reply compiler/imgdata2hdf5/README.md]: I've changed the message to `See docs/how-to-prepare-virtualenv.txt`

File: compiler/record-minmax/src/HDF5Importer.cpp [hunk 2]
```diff
@@ -62,6 +62,10 @@ DataType toInternalDtype(const H5::DataType &h5_type)
   return DataType::Unknown;
 }
 
+void readTensorData(H5::DataSet &tensor, uint8_t *buffer)
+{
+  tensor.read(buffer, H5::PredType::NATIVE_UINT8);
+}
```
Comment (seanshpark) [inline compiler/record-minmax/src/HDF5Importer.cpp:68]: may be add empty line after this?

Comment (jinevening) [reply compiler/record-minmax/src/HDF5Importer.cpp:68]: There are multiple `readTensorData`, so I placed them together for readability.

**Pull request closed without merging**